tools/screenshots/konnect/mesh-manager/mesh-manager-dashboard.yaml

```diff
@@ -0,0 +1,8 @@
+- output: ../../app/assets/images/products/konnect/mesh-manager/konnect-mesh-control-plane-dashboard.png
+  url: https://cloud.konghq.com/us/mesh-manager/2b690e35-3334-4cf1-b172-047582ed3fa4
+  wait: 4000
```

Comment:
@Guaris So, it turns out the wait time was necessary. When I removed it or did `1000`, it didn't load the page in time. Though I think `2000` might work.

I set up the Mesh in our team docs account, so you can look at it. For me, it says the proxies and services are online (all 4/4 of them). However, in the shot scraper screenshot, it was showing 0/4 services and proxies connected. Any idea why that might be? 
<img width="402" alt="Screenshot 2023-10-05 at 3 23 13 PM" src="https://github.com/Kong/docs.konghq.com/assets/75819066/11be05f4-b684-4c49-9309-23e76f9f9448">

Reply:
I'm getting 
0/4 for all of them too

![image](https://github.com/Kong/docs.konghq.com/assets/23319190/a8f02e0e-00a7-4ac8-b9f3-ca74c79ed060)

Reply:
@Guaris Yeah, I'm not sure why they're all disconnected now. They were online when I added them. I think this PR is still okay to be reviewed and merged if everything looks good and I can figure out why they are offline later.

Reply:
Agreed.